internal/cli/organizations.go

```diff
@@ -109,26 +113,57 @@ func listOrganizationsCmd(cli *cli) *cobra.Command {
 		Long: `List your existing organizations. To create one try:
 auth0 orgs create`,
 		Example: `auth0 orgs list
-auth0 orgs ls`,
+auth0 orgs ls
+auth0 orgs ls -n 100`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var list *management.OrganizationList
-
-			if err := ansi.Waiting(func() error {
-				var err error
-				list, err = cli.api.Organization.List()
+			orgs, err := cli.getOrgsWithSpinner(cmd.Context(), inputs.Number)
+			if err != nil {
 				return err
-			}); err != nil {
-				return fmt.Errorf("An unexpected error occurred: %w", err)
 			}
 
-			cli.renderer.OrganizationList(list.Organizations)
+			cli.renderer.OrganizationList(orgs)
 			return nil
 		},
 	}
-
+	number.RegisterInt(cmd, &inputs.Number, defaultPageSize)
 	return cmd
 }
 
+func (cli *cli) getOrgsWithSpinner(context context.Context, number int) ([]management.Organization, error) {
+	var orgs []management.Organization
+	err := ansi.Spinner("Getting organizations", func() error {
+		var errInner error
+		orgs, errInner = cli.getOrgs(context, number)
+		return errInner
+	})
+	return orgs, err
+}
+
+func (cli *cli) getOrgs(context context.Context, number int) ([]management.Organization, error) {
+	list, err := getWithPagination(
+		context,
+		number,
+		func(opts ...management.RequestOption) (result []interface{}, hasNext bool, err error) {
+			orgs, apiErr := cli.api.Organization.List(opts...)
+			if apiErr != nil {
+				return nil, false, apiErr
+			}
+			var output []interface{}
+			for _, org := range orgs.Organizations {
+				output = append(output, *org)
+			}
+			return output, orgs.HasNext(), nil
+		})
+	if err != nil {
+		return nil, fmt.Errorf("Unable to list organizations: %w", err)
+	}
+	var typedList []management.Organization
+	for _, item := range list {
+		typedList = append(typedList, item.(management.Organization))
+	}
+	return typedList, nil
```

Comment:
Could we please follow the same style for these funcs that was introduced in https://github.com/auth0/auth0-cli/pull/450/files 🙏🏻 ?

Reply:
Sure, I can remove `getOrgsWithSpinner` and `getOrgs`. Do you want me to keep the `ansi.Spinner`? It is not used in `listRolesCmd` you linked, though.

Reply:
I addressed your comment by using the same func in `roles` list command. I discarded the `ansi.Spinner` to keep it consistent with `roles list` user experience.